epochStart/bootstrap/common.go

```diff
@@ -0,0 +1,25 @@
+package bootstrap
+
+import (
+	"time"
+
+	"github.com/ElrondNetwork/elrond-go/config"
+	"github.com/ElrondNetwork/elrond-go/sharding"
+)
+
+// ShouldSyncWithTheNetwork returns true if a peer is not synced with the latest epoch (especially used when a peer
+// wants to join the network after the genesis)
+func ShouldSyncWithTheNetwork(
```

Comment:
i think we need a factory, which created the epoch start bootstrapper. than it is launched and resolved in all cases.

Reply:
done